rtf_vis_tool/src/tests/SE3.test.js

```diff
@@ -0,0 +1,31 @@
+/* Unit Test for the SE3 Class.
+
+Author: Adi Singh
+*/
+
+// Third-Party Package Imports.
+var nj = require('numjs');
+
+// Local Imports.
+var SE3 = require('../Components/frustum_classes/se3');
+
+describe('SE3 Class Unit Tests', () => {
+
+    /* Creates a simple Pose matrix that flips x and z coordinates and translates by [2,3,4]. 
+       Applies to the sample point [1,2,3] and verifies the SE3 transformation result is accurate.
+    */
+    it('Test transform_from.', () => {
+        const rotation = nj.array([[0, 0, 1],

```

Comment:
maybe a comment could explain that this rotation matrix corresponds to Rx, Ry, Rz of [180, 0, 180] degrees?